### PR TITLE
added validation to subject component and created method onSubmit

### PR DESCRIPTION
**Pull request by Rihovardini**

### CODE REVIEW

File: src/app/admin-panel/admin-panel/subjects/subjects.component.ts [hunk 1]
```diff
@@ -1,77 +1,132 @@
...
+
+  cols: Array<object>;
+
+  constructor(
+    private _subjectsService: AdminSubjectsService,
+    private formBuilder: FormBuilder
+  ) {}
+
+  ngOnInit() {
+    this.loading = true;
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/subjects/subjects.component.ts]: where is this used?

File: src/app/admin-panel/admin-panel/subjects/subjects.component.ts [hunk 2]
```diff
@@ -1,77 +1,132 @@
...
+      {
+        field: "subjectName",
+        header: "Назва"
+      },
+      {
+        field: "subjectDescription",
+        header: "Опис"
+      }
+    ];
+    this.loading = false;
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/subjects/subjects.component.ts]: i believe it must be inside subscription

Comment (Rihovardini) [reply src/app/admin-panel/admin-panel/subjects/subjects.component.ts]: Yeah, you are right. 
I replaced it.
Thx)

File: src/app/admin-panel/admin-panel/subjects/subjects.component.ts (hunk 1 above)
Comment (Rihovardini) [reply src/app/admin-panel/admin-panel/subjects/subjects.component.ts]: It's for loading spinner

**Pull request merged**